schemas/2015-10-01/Microsoft.DataFactory.json

```diff
@@ -125,7 +125,7 @@
         }
       }
     },
-    "pipelines": {
+    "dataPipelines": {

```

Comment:
you need to update the 2 deploymentTemplate.json files for this reference name change.